SPECS/mozjs60/mozjs60.spec

```diff
@@ -64,6 +64,7 @@ popd
 rm -rf ../../modules/zlib
 
 %build
+mkdir -pv $(readlink /dev/shm)
```

Comment:
need to bump the release number for spec file changes.

Reply:
so for instance if the line said this:
Release:       9%{?dist}
then I'll bump it to Release: 10%{?dist}?